src/state/slices/portfolioSlice/utils/index.ts

```diff
@@ -53,8 +53,8 @@ import type {
   Portfolio,
   PortfolioAccountBalancesById,
   PortfolioAccounts as PortfolioSliceAccounts,
-} from './portfolioSliceCommon'
-import { initialState } from './portfolioSliceCommon'
+} from '../portfolioSliceCommon'
+import { initialState } from '../portfolioSliceCommon'
 
 // note - this isn't a selector, just a pure utility function
 export const accountIdToLabel = (accountId: AccountId): string => {
```

Comment:
should probably move `utils.test.ts` inside `utils/` as well

Reply:
dbc66bde92